.circleci/config.yml

```diff
@@ -292,32 +311,50 @@ jobs:
     steps:
       - checkout
       - run: REPO_DOMAIN=kubeapps REPO_NAME=kubeapps ./script/create_release.sh ${CIRCLE_TAG}
-  local_e2e_tests_mongodb:
+  local_e2e_tests_mongodb_helm2:
+    machine: true
+    environment:
+      KUBEAPPS_DB: "mongodb"
+      HELM_VERSION: "v2.14.0"
```

Comment:
Just out of interest, why not 2.16?

Reply:
We try to use the oldest version possible. Helm always maintains backward compatibility so if our chart works with an older version, it will work with newer ones (but not the other way around).